sqrl-planner/sqrl-planner-local/src/test/resources/snapshots/com/datasqrl/flink/FlinkPhysicalIT/fuzzedValuesTest.txt

```diff
@@ -0,0 +1,255 @@
+precision-loss-leading-decimal-order, 2023-05-19T02:16:39.553-07:00
+precision-loss-trailing-decimal-order, 2023-05-19T02:16:39.553-07:00
+simple-single-item-order, 2023-05-19T01:45:39.553-07:00
+simple-two-item-order, 2023-05-19T01:29:39.553-07:00
+simple-two-item-order-same-customer, 2023-05-19T02:16:39.553-07:00
+>>>distinctorderstime-data
+order-with-decimal, 2023-06-19T01:29:39.553-07:00
+order-with-future-date, 2025-06-19T01:29:39.553-07:00
+order-with-max-int-quantity, 2023-05-19T02:28:39.553-07:00
+order-with-multiple-entries, 2023-06-19T02:29:39.553-07:00
+order-with-null-discount, 2023-05-19T02:28:39.553-07:00
+order-with-variety-of-values, 2023-06-19T01:29:39.553-07:00
+precision-loss-leading-decimal-order, 2023-05-19T02:16:39.553-07:00
+precision-loss-trailing-decimal-order, 2023-05-19T02:16:39.553-07:00
+simple-single-item-order, 2023-05-19T01:45:39.553-07:00
+simple-two-item-order, 2023-05-19T01:29:39.553-07:00
+simple-two-item-order-same-customer, 2023-05-19T02:16:39.553-07:00
+>>>orders-data
+order-with-decimal, 1000150, 2023-06-19T01:29:39.553-07:00
+order-with-future-date, 1000153, 2025-06-19T01:29:39.553-07:00
+order-with-max-int-quantity, 1000107, 2023-05-19T02:28:39.553-07:00
+order-with-multiple-entries, 1000151, 2023-06-19T02:29:39.553-07:00
+order-with-null-discount, 1000107, 2023-05-19T02:28:39.553-07:00
+order-with-past-date, 1000154, 2021-06-19T01:29:39.553-07:00
+order-with-variety-of-values, 1000150, 2023-06-19T01:29:39.553-07:00
+precision-loss-leading-decimal-order, 1000121, 2023-05-19T02:16:39.553-07:00
+precision-loss-trailing-decimal-order, 1000121, 2023-05-19T02:16:39.553-07:00
+simple-single-item-order, 1000107, 2023-05-19T01:45:39.553-07:00
+simple-two-item-order, 1000101, 2023-05-19T01:29:39.553-07:00
+simple-two-item-order-same-customer, 1000121, 2023-05-19T02:16:39.553-07:00
+>>>entries-data
+order-with-decimal, 0, 1, 10, 0, null, 2023-06-19T01:29:39.553-07:00
+order-with-decimal, 1, 2, 20, 1000000000000.12341309, 0, 2023-06-19T01:29:39.553-07:00
+order-with-decimal, 2, 3, 30, -1000000000000.12341309, 0, 2023-06-19T01:29:39.553-07:00
+order-with-decimal, 3, 4, 40, 5E+2, 9E+2, 2023-06-19T01:29:39.553-07:00
+order-with-decimal, 4, 5, 50, 1E+2, -5E+1, 2023-06-19T01:29:39.553-07:00
+order-with-decimal, 5, 6, 1000000, 1, 0, 2023-06-19T01:29:39.553-07:00
+order-with-future-date, 0, 1, 10, 0, null, 2025-06-19T01:29:39.553-07:00
+order-with-future-date, 1, 2, 2, -1E+2, 0, 2025-06-19T01:29:39.553-07:00
+order-with-future-date, 2, 3, 30, 2, 0, 2025-06-19T01:29:39.553-07:00
+order-with-max-int-quantity, 0, 1332, 8, 8.49, null, 2023-05-19T02:28:39.553-07:00
+order-with-max-int-quantity, 1, 3571, 2147483647, 41.95, 5, 2023-05-19T02:28:39.553-07:00
+order-with-multiple-entries, 0, 1, 1, 1E+1, 0, 2023-06-19T02:29:39.553-07:00
+order-with-multiple-entries, 1, 2, 1, 1E+1, 0, 2023-06-19T02:29:39.553-07:00
+order-with-multiple-entries, 10, 11, 1, 1E+1, 0, 2023-06-19T02:29:39.553-07:00
```

Comment:
Why is it doing 1E+1 instead of 10?

Reply:
Ah my mistake, it's a consequence of stripping trailing 0s. I'll fix.